plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/transactionlog/checkpoint/TestCheckpointEntryIterator.java

```diff
@@ -87,8 +87,8 @@ public void testReadMetadataEntry()
                 .isEqualTo(
                         new MetadataEntry(
                                 "b6aeffad-da73-4dde-b68e-937e468b1fde",
-                                "",
-                                "",
+                                null,
```

Comment:
is "name" a table name? shouldn't it be present?

Reply:
It's optional "User-provided identifier for this table" according to https://github.com/delta-io/delta/blob/master/PROTOCOL.md#change-metadata. 

I can send another PR if you agree on setting the field at https://github.com/trinodb/trino/blob/master/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java#L1004